src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+
+const SEX_MALE = 'm';
+const SEX_FEMALE = 'f';
+const CENTURY_CHANGE = 100;
+const INITIAL_VALUE = 0;
+
+function getAverageAge(people) {
+  const totalAge = people.reduce((sum, person) =>
```

Comment:
you don't need variable totalAge you already can return  people.reduce( ... ) /  people.length;